### DISCUSSION TITLE
printing starts in the middle and runs off bed

### DISCUSSION THREAD

**Discussion by Hr46ph** [Jyers/Marlin#1926] [Q&A]
Ive been printing for months with good succes on stock firmware. Manual bed level (no bltouch) and always get decent first layers but my bed is warped and its visible. I figured I'd try the bltouch that came with my printer. I installed it, I flashed Jyers and set Z offset, did a manual bed level on the the four corners to 0.1mm and ran the mesh. Saved it. Next I configured Octoprint following the wiki and send a first layer test print without filament to measure if I need a Z offset in my slicer. It homes to the middle, then lowers, and runs off the bed to the right. It doesnt start at bottom left corner, but starts in the center as if that was the bottom right hand corner.

I've went over the wiki several times, I searched, and went over all the printer settings.

I cant figure this out, what is going on?

I own a Ender 3 v2, 4.2.2 board with Microswiss hotend and bltouch.

Thanks for the help.

**Comment by SikVac** [comment]
First after Flashing the firmware, do: 
- Reset_to_Default, 
- following by a Store Settings 
- and following by a Restore Settings

**Comment by Hr46ph** [comment]
Thanks for the reply, and Im sorry I didn't mention that, but I did a reset and store, followed by a power off/power on.

I didn't do the restore, but I'll go through it once more just to be sure.

**Comment by Hr46ph** [comment]
Oke, I completely started over, resetting, storing, restore and power cycles. It does exactly the same.

I start a print it goes to the middle to level and moves off the bed to the right.

**Reply by Hr46ph** [reply]
When I use the manual mesh version and disconnect bltouch it works fine, I just need to configure X/Y offset in the slicer so it doesnt start in front of the bed after homing. Using the same print (a simple 1 layer bed level test) that prints correctly on the jyers manual mesh version still starts in the middle of the bed after homing/probing, and runs off the bed.

I tried the official firmware with bltouch support from Creality, and it does the same thing.

I've use the bltouch before using the smith3d firmware (before they switched to Jyers) and that worked fine.

Im at a loss ...

**Reply by hahmlet** [reply]
Hey @Hr46ph did you ever find a solution? I have exact setup Ender 3V2, 4.2.2, Microswiss hotend. Only difference is CR touch instead of BL touch. Everything works great except the print starts in the center of the plate and the nozzle wipe immediately runs off the plate to the right.

When I Home X axis, it homes all the way to the left. Homing Y axis homes it to the bottom. Then homing Z and it zooms to the center, homes in the center.

**Reply by Hr46ph** [reply]
No, I haven't. Every open source firmware I tried did the same thing and everywhere I asked for help there were hardly any replies. It must be configuration related though ... I gave up. And I am glad I did because this led me to take the dive into Klipper with FluidD on RPi. I should have done that right away and I'll never go back. It's divine. I suggest you at least look into that as well.

**Reply by hahmlet** [reply]
@Hr46ph I've gone for its-so-stupid-I-can't-believe-it-worked.

I just homed X and Y again (BUT NOT Z) after it's done all its homing and prep. Starts the print exactly where it should be.

G28 ; home all axes
M420 S1 ; apply mesh
M104 S215 ; heat hotend and continue
M190 S60 ; Heat bed and wait
**G28 X ; Home X
G28 Y ; Home Y**
G92 X-5.0 Y-10.0 ; set home offsets
G1 X0 Y0 Z0.2 F3000 ; get ready to prime
G92 E0 ; reset extrusion distance
G1 X160 E15 F600 ; prime nozzle
G1 X180 F5000 ; quick wipe

**Reply by Hr46ph** [reply]
Glad it works for you and provided an answer for others to find.
